oam/install_scripts/mcs-loadbrm.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import configparser
+import subprocess
+import xml.etree.ElementTree as ET
```

Comment:
I would prefer having lxml here b/c Columnstore.xml will have external entities that xml parser from the standart packages doesn support.